app/assets/stylesheets/application.css.scss.erb

```diff
@@ -14,22 +14,21 @@
 td, th {
   padding: 15px 5px;
   display: table-cell;
-  text-align: -webkit-center;
   vertical-align: middle;
   border-radius: 2px;
 }
 
 .manage_btn {
     border: none;
-    border-radius: 2px;
-    display: inline-block;
-    height: 36px;
-    outline: 0;
-    padding: 0 1rem;
-    text-transform: capitalize;
-    vertical-align: middle;
-    background-color: #00695c;
-    color: #fff;
+    // border-radius: 2px;
```

Comment:
I'm not sure if these should be commented out or deleted. I guess you have your reasons.